crates/ruma-events/src/room/language.rs

```diff
@@ -0,0 +1,75 @@
+//! Types for the [`m.room.language`] event.
+//!
+//! [`m.room.language`]: https://github.com/matrix-org/matrix-spec-proposals/pull/4334
+
+use ruma_macros::EventContent;
+use serde::{Deserialize, Serialize};
+
+use crate::EmptyStateKey;
+
+/// The content of an `m.room.language` event.
+///
+/// The room language is a [IETF BCP 47](https://developer.mozilla.org/en-US/docs/Glossary/BCP_47_language_tag) language code.
+#[derive(Clone, Debug, Deserialize, Serialize, EventContent)]
+#[cfg_attr(not(ruma_unstable_exhaustive_types), non_exhaustive)]
+#[ruma_event(type = "org.matrix.msc4334.room.language", kind = State, state_key_type = EmptyStateKey)]
+pub struct RoomLanguageEventContent {
+    /// The language of the room.
+    pub language: String,
+}
+
+impl RoomLanguageEventContent {
+    /// Create a new `RoomLanguageEventContent` with the given language.
+    pub fn new(language: String) -> Self {
+        Self { language }
+    }
+}
+
+impl Default for RoomLanguageEventContent {
+    fn default() -> Self {
+        Self { language: String::from("en-US") }
```

Comment:
This default is not documented in the MSC, so it shouldn't be implemented.

I would also argue that it doesn't make sense to assume a default, so it should be a choice of the implementation if it always needs a language somewhere.